doc/source/multivariate.rst

```diff
@@ -144,12 +144,12 @@ We realize that the mean vector and the covariance often have special forms in p
 
 .. code-block:: julia
 
-    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector{Float64},ZeroVector{Float64}}} <: AbstractMvNormal
+    immutable MvNormal{Cov<:AbstractPDMat,Mean<:Union{Vector,ZeroVector}} <: AbstractMvNormal
```

Comment:
Yes. Typically it's the first argument so maybe follow that convention here as well. I think we should restrict μ and Σ to have the same element type to simplify things. When would it be useful to allow for that?